### ISSUE TITLE
Enable ppc64le qemu

### ISSUE THREAD

**Issue by shenki** [free-electrons/toolchains-builder#8]
I'd like to enable boot tests for the ppc64le toolchains. buildroot master has support for a qemu_ppc64le_pseries machine as of 9d5b5b7bfb1d8cd3c12067650a8d2bf1c877a422.

Here's a change that enables the configuration:

 https://github.com/shenki/toolchains-builder/tree/enable-ppc64le

However I think we need to wait until the toolchains-builder uses a newer release of buildroot?

I'm happy to write and test patches with some guidance as to what needs to be done. Thanks!

**Comment by tpetazzoni** [comment]
Thanks for this contribution, very useful!

No, we don't necessarily need to wait until toolchains-builder uses a newer release of Buildroot. toolchains-builder uses a Buildroot fork at https://github.com/free-electrons/buildroot-toolchains, with one branch for the stable toolchains and one branch for the bleeding edge toolchains. And those branches containing some backports/fixes, and therefore we can backport your ppc64le defconfig patch into those branches.

**Comment by tpetazzoni** [comment]
Sorry for the slow reply. Since your commit, I've changed build.sh quite significantly, but I've adapted your change. I've also done some changes to board/qemu/ppc64le-pseries/readme.txt to make the Qemu command line more similar to the one in board/qemu/ppc64-pseries/readme.txt, and I've fired up a test build at https://gitlab.com/free-electrons/toolchains-builder/pipelines/18271434. We'll see what it gives. If it works fine, I'll apply, and submit the necessary patches to upstream Buildroot.
